### DISCUSSION TITLE
Long break Announcement

### DISCUSSION THREAD

**Discussion by sylouuu** [furybee/chrome-tab-modifier#272] [Announcements]
Hello Tab Modifier lovers :D

Some stats about the project:

* First release: **2014**
* Commits: **500+**
* Releases: **53**
* Contributors: **7**
* Current users: **30k+** (Chrome & Opera)

The last update was a year ago. As you can see, there are many open issues and pull requests. I don't want to spend time on this project anymore. I passively decided to leave Tab Modifier "as is".  I may or may not update TM in the future.

I'm not searching for new owner/contributor at the moment. If you want to tweak TM, I suggest you to fork the project and use your modified TM version locally.

Thanks for your understanding. Consider this post as a *long* break!

Regards

Syl

**Comment by MartinLichtblau** [comment]
Sad to hear. But yeah, life goes on.

**Comment by brakedalen** [comment]
Any chance that someone can port this to safari? Love the extension, and desperately need it on safari 😬
It looks simple if you a familiar with coding:
https://developer.apple.com/documentation/safariservices/safari_web_extensions
